tests/test_main.py

```diff
@@ -59,98 +65,57 @@
-    with open(metadata, "rt") as fn:
-        eq_("{phred-offset: 33}", fn.readline().replace("\n", ""))
-
-
-def test_fastq_id_maker():
-    exp1, exp2 = _itsxpress._fastq_id_maker(per_sample_sequences=TEST_DATA,
-                                            artifact_type=ARTIFACT_TYPE_P)
-    for sequence in exp1:
-        eq_(sequence["paths"][0], '4774-1-MSITS3_0_L001_R1_001.fastq.gz')
-        eq_(sequence["paths"][1], '4774-1-MSITS3_1_L001_R2_001.fastq.gz')
-        eq_(exp2, False)
-    raises(ValueError, lambda: _itsxpress._fastq_id_maker(per_sample_sequences=TEST_DATA_PAF,
-                                                          artifact_type=ARTIFACT_TYPE_P))
-    exp3 = _itsxpress._fastq_id_maker(per_sample_sequences=TEST_DATA,
-                                      artifact_type=ARTIFACT_TYPE_S)
-    eq_(exp3[1], True)
-
-
 def test_taxa_prefix_to_taxa():
     exp1 = _itsxpress._taxa_prefix_to_taxa(taxa_prefix="A")
     eq_(exp1, "Alveolata")
 
 
-def test_set_fastqs_and_check():
-    sequences, single_end = _itsxpress._fastq_id_maker(per_sample_sequences=TEST_DATA,
-                                                       artifact_type=ARTIFACT_TYPE_P)
-    for sequence in sequences:
-        exp1 = _itsxpress._set_fastqs_and_check(per_sample_sequences=TEST_DATA,
-                                                artifact_type=ARTIFACT_TYPE_P,
-                                                sequence=sequence,
-                                                single_end=single_end,
-                                                threads=1)
-        eq_(exp1[0], "4774-1-MSITS3")
-
-    sequences2, single_end2 = _itsxpress._fastq_id_maker(per_sample_sequences=TEST_DATA_PAF,
-                                                         artifact_type=ARTIFACT_TYPE_S)
-    for sequence2 in sequences2:
-        exp2 = _itsxpress._set_fastqs_and_check(per_sample_sequences=TEST_DATA,
-                                                artifact_type=ARTIFACT_TYPE_S,
-                                                sequence=sequence2,
-                                                single_end=single_end2,
-                                                threads=1)
-        eq_(exp2[0], "4774-1-MSITS3")
-
-    sequences3= _itsxpress._fastq_id_maker(per_sample_sequences=TEST_DATA_PAF,
-                                           artifact_type=ARTIFACT_TYPE_S)
-    for sequence3 in sequences3[0]:
-        exp3 = _itsxpress._set_fastqs_and_check(per_sample_sequences=TEST_DATA,
-                                                artifact_type=ARTIFACT_TYPE_S,
-                                                sequence=sequence3,
-                                                single_end=True,
-                                                threads=1)
-        eq_(exp3[0], "4774-1-MSITS3")
-
-
-def test_trim_pair():
-    threads = 1
-    taxa = "F"
-    region = "ITS2"
+class SetFastqsAndCheckTests(unittest.TestCase):
```

Comment:
Please double-check me here, I am not 100% I captured the original intent of these tests.